client/app/api/course/Assessment/Submissions.js

```diff
@@ -130,10 +130,9 @@ export default class SubmissionsAPI extends BaseAssessmentAPI {
   }
 
   fetchLiveFeedback(feedbackUrl, feedbackToken) {
-    return this.externalClient.get(`/signed/v2/feedback/llm`, {
+    return this.externalClient.get(`/signed/feedback/llm`, {
       baseURL: feedbackUrl,
-      // TODO remove this when we cutover to live CV instance
-      headers: { 'ngrok-skip-browser-warning': '1' },
+      headers: { 'x-api-version': '2.1' },
```

Comment:
Nit: Should set a constant then use it. Especially when we may eventually to use for other things like problem generation.   
Set to the same name as backend so future devs can find the constant easily.

Reply:
done